src/includes/user-feedback/example-widget/dotnet.mdx

```diff
@@ -0,0 +1,24 @@
+Make sure you've got the JavaScript SDK available:
+
+```html
+<script
+  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
+  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
+  crossorigin="anonymous"
+></script>
+```
+
+You'll then need to call `showReportDialog` and pass in the generated event ID.
+This event ID is returned from all calls to <PlatformIdentifier
+name="capture-event" /> and <PlatformIdentifier name="capture-exception" />.
+There is also a function called <PlatformIdentifier name="last-event-id" />
+that returns the ID of the most recently sent event.
+
```

Comment:
Can we add a `note` that for users using ASP.NET or ASP.NET Core, there's a specific content for this?
We have overrides for this on those that include a proper snippet:

https://docs.sentry.io/platforms/dotnet/guides/aspnet/enriching-events/user-feedback/#integration

https://docs.sentry.io/platforms/dotnet/guides/aspnetcore/enriching-events/user-feedback/#integration

I'm just afraid folks land on .NET and click on User Feedback on the left without understanding that there's a "better" content if they are in fact using a web technology.

Reply:
yep. The design does that now. 

ASP.NET - https://sentry-docs-kl81mjezf.sentry.dev/platforms/dotnet/guides/aspnet/enriching-events/user-feedback/#integration


ASP.NET Core - https://sentry-docs-kl81mjezf.sentry.dev/platforms/dotnet/guides/aspnetcore/enriching-events/user-feedback/#integration

Reply:
I meant that in here: https://sentry-docs-kl81mjezf.sentry.dev/platforms/dotnet/enriching-events/user-feedback/

There's no indication that this snippet:
![image](https://user-images.githubusercontent.com/1633368/116304376-82595380-a770-11eb-9bdf-46342c685438.png)

Is not what you need if you're on ASP.NET or ASP.NET Core (90% of all Web Apps in .NET fall into one of these two)

Reply:
ah, got it. Yes, we can do that.